src/messages/interactive.js

```diff
@@ -15,7 +15,7 @@ class Interactive {
      * Create an Interactive object for the API
      *
      * @param {(ActionList|ActionButtons|ActionCatalog)} action The action component of the interactive message
-     * @param {Body} [body] The body component of the interactive message
```

Comment:
Keep in mind, body CAN be optional if the "action is an ActionCatalog with a single product"

```js
// This is valid
new Interactive(
   new ActionCatalog(
      new Product("whatever parameters")
   )
   // No body required
);
```

It's not the neatest solution, ActionCatalog was hard to implement with all its exceptions.

Reply:
I saw this case with "Image caption" in Header, but then WhatsApp APIs stopped it. So we can leave it the same here.

What do you think?

Reply:
Sorry, I didn't follow. What's up with image captions in Header?

Anyway, the optional is required if you want to get JSCheck or TS checks in your code.

With optional
![image](https://user-images.githubusercontent.com/46955459/217936117-5dc58ba8-5800-4303-bb0b-9e62f31dd7e4.png)

Without optional
![image](https://user-images.githubusercontent.com/46955459/217936033-55f1c9b6-ad29-4483-8dc1-330386e1f35c.png)
(Translation: Expected 2-4 arguments, but got 1. No argument provided for body)